catroid/src/main/res/values-ar/strings.xml

```diff
@@ -1447,4 +1447,14 @@
   <string name="crash_dialog_always_send">دائماً أرسل</string>
   <string name="crash_dialog_send">أرسل</string>
   <!--  -->
+
+  <!--  -->
+  <string name="Byte_short">بايت</string>
+  <string name="kiloByte_short">كيلوبايت</string>
```

Comment:
Looking really good!
I'd just write "kiloByte_short" with an uppercase "K" for consistencies sake. Is there a reason you left it like this?
(The same thing should be done in the default values/strings.xml)